Makefile

```diff
@@ -92,7 +92,7 @@ install_fasm2bels:
 	$(IN_ENV) cd third_party/fasm2bels && make test-py
 
 # Run a simple design through fasm2bels to generate the database.
-	$(IN_ENV) python scripts/run_design.py designs/basic/and3/ xilinx_and_reversed
+	$(IN_ENV) python scripts/scripts_legacy/run_design.py designs/basic/and3/ xilinx_and_reversed
```

Comment:
Let's use run.py for this.

Reply:
@jgoeders How do you use run.py to run that same flow? Or does it not matter which flow/design is used here?

Reply:
`xilinx_and_reversed` turned into `VivadoBitAnalysis`

Reply:
@yonnorc42 just for personal reference, the flow used here does matter in that it has to use f2b. The first time f2b runs, it generates a database for the part and its bitstream, which is used on subsequent runs of the tool.